array-methods-task7.js

```diff
@@ -12,5 +12,14 @@ let mary = { name: "Mary", age: 28 };
 let users = [john, pete, mary];
 
 // Own code
-let names = [john.name, pete.name, mary.name].join(", ");
-// console.log(names); // John, Pete, Mary
+// Not generic enough
+// let names = [john.name, pete.name, mary.name].join(", ");
+function arrayOfNames(array) {
+  for (i = 0; i < array.length; i++) {
+    let newArray = array[i].name;
```

Comment:
Why are you creating a variable called `newArray` when in fact it's the name of the item?